app/src/main/res/layout/activity_main.xml

```diff
@@ -1,24 +1,42 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout
+<layout
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:bind="http://schemas.android.com/tools"
     xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:background="@drawable/main_theme"
-    tools:context="com.framgia.pokemon.screen.main.MainActivity"
     >
-    <FrameLayout
-        android:layout_width="wrap_content"
-        android:layout_height="@dimen/dp_0"
-        android:id="@+id/frame_layout"
-        />
-    <android.support.design.widget.BottomNavigationView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_gravity="bottom"
-        android:background="?android:attr/windowBackground"
-        app:menu="@menu/navigation"
-        app:itemBackground="@color/colorAccent"
-        />
-</LinearLayout>
+
+    <data>
+
+        <variable
+            name="viewModel"
+            type="com.framgia.pokemon.screen.main.MainViewModel"/>
+    </data>
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@drawable/main_theme"
+        tools:context="com.framgia.pokemon.screen.main.MainActivity"
+        >
+
+        <android.support.v4.view.ViewPager
+            android:id="@+id/view_pager"
+            android:layout_width="match_parent"
```

Comment:
WARNING: [InefficientWeight] Use a `layout_width` of `0dp` instead of `match_parent` for better performance
 When only a single widget in a LinearLayout defines a weight, it is more efficient to assign a width/height of `0dp` to it since it will absorb all the remaining space anyway. With a declared width/height of `0dp` it does not have to measure its own size first.